src/descriptors/structs.json

```diff
@@ -179,8 +179,8 @@
     "supertype": "Forecast"
   },
   {
-    "struct_name": "CostCoefficient",
-    "docstring": "A forecast for cost function data field in a Component.",
+    "struct_name": "PiecewiseCost",
```

Comment:
This works for any `PieceWiseFunction` not just cost.

Reply:
will do